libcontainer/logs/logs_linux_test.go

```diff
@@ -11,44 +11,51 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const msgErr = `"level":"error"`
+
 func TestLoggingToFile(t *testing.T) {
 	l := runLogForwarding(t)
 
-	logToLogWriter(t, l, `{"level": "info","msg":"kitten"}`)
+	msg := `"level":"info","msg":"kitten"`
+	logToLogWriter(t, l, msg)
```

Comment:
Curly braces are now added by `LogToLogWriter`. This is done because otherwise we'll have to remove (at least the) closing brace in `check()`. We send the message as `{"level": "info","msg":"kitten"}`, but the logrus adds the timestamp so if `msg` contains the curly braces, the test fails like this:
```
    logs_linux_test.go:22: {"level":"info","msg":"kitten","time":"2021-08-17T11:02:02-07:00"}
         does not contain {"level":"info","msg":"kitten"}
```

IOW I want the `msg` to be good enough for consumption by both `logToLogWriter()` and `check()`, and it's easier to add braces in `logToLogWriter` than to remove them in `check()`.